03 Writing Algorithms/03 Securities/99 Asset Classes/07 Futures/04 Market Hours/14 COMEX/03 market-opening-hours.php

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
The name of this file should be `03 Writing Algorithms/03 Securities/99 Asset Classes/07 Futures/04 Market Hours/14 COMEX/03 Regular Trading Hours.php`. See https://github.com/QuantConnect/Documentation/issues/1200